src/yarpWholeBodyStates.cpp

```diff
@@ -760,7 +768,37 @@ bool yarpWholeBodyEstimator::threadInit()
     left_sole_local_id = wbi::ID(LEFT_LEG,8);
     right_sole_local_id = wbi::ID(RIGHT_LEG,8);
     */
-
+    
+    /*    Eigen::Map<Eigen::VectorXd> dqjVect(dqj,dof);
+    Eigen::Matrix<double,6,Eigen::Dynamic,Eigen::RowMajor> complete_jacobian(6,dof+6), joint_jacobian(6,dof),floatingBase_jacobian(6,6);
+    Eigen::Matrix<double,6,Eigen::Dynamic,Eigen::RowMajor> tempMatForComputation(6,dof);//, minusTemp(6,dof);
+    Eigen::VectorXd dvbVect(6);
+    Eigen::Map<Eigen::VectorXd> yarpWrapper(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
+  */  
+  
+     int dof = estimates.lastQ.length();
+//     dqjVect_.resize(6,dof+6);
+//     std::cout<<" dqjVect : "<<dqjVect_.rows()<<" , "<<dqjVect_.cols()<<"\n"; 
+
+
+    
+    complete_jacobian.resize(6,dof+6);
+    joint_jacobian.resize(6,dof);
+//     floatingBase_jacobian(6,6);
+    tempMatForComputation.resize(6,dof);
+    new (&rotationalVelocityWrapper) Eigen::Map<Eigen::VectorXd>(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
+    new (&dqjVect) Eigen::Map<Eigen::VectorXd>(estimates.lastDq.data(),dof);
```

Comment:
What this `new (&a) b` is meaning? The `Eigen::Map` object do not allocate memory (they are meant for using memory allocated by someone else) so I guess it is ok to have them as local variables.